backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5700,4 +5696,93 @@ databaseChangeLog:
         rollback:
           - sql:
               sql: DELETE FROM ${database.defaultSchemaName}.supported_disease WHERE name = 'Gonorrhea';
-
+  - changeSet:
+      id: create-facility_providers-table
+      author: bobby@skylight.digital
+      changes:
+        - tagDatabase:
+            tag: add-facility-providers-table
+        - createTable:
+            tableName: facility_providers
+            remarks: Join table for mapping multiple providers to a facility.
+            columns:
+              - column:
+                  name: facility_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__facility_providers__facility
+                    references: facility
+              - column:
+                  name: provider_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__facility_providers__provider
+                    references: provider
+  - changeSet:
```

Comment:
this change is just so we can view the new table from metabase right?

Reply:
yes that's right!

Reply:
oh uh oh it doesn't seem to have worked though now that I'm looking at dev4 metabase. investigating...